dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/net/RequestUniqueIdentifier.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.core.internal.net
+
+import okhttp3.Request
+
+/**
+ * Generates an identifier to uniquely track requests.
+ */
+internal fun identifyRequest(request: Request): String {
```

Comment:
Shouldn't we use this method also in the TracingRequestInterceptor to avoid the issue we have there with the unique requests ids into the HashMap ?

Reply:
Yes but since there's another ticket for that I didn't want to mix tasks.